tox.ini

```diff
@@ -65,16 +73,15 @@ allowlist_externals =
 ;deps =
 ;extras =
 
-[testenv:notebooks_doctests{-coverage,}]
-description = Run notebooks and doctests with pytest under {basepython}
+[testenv:notebooks{-prefetch,}]
+description = Run notebooks with pytest under {basepython}
 commands =
     pytest --no-cov --nbval --dist=loadscope --rootdir=tests/ --ignore=docs/notebooks/example.ipynb docs/notebooks
-    pytest --rootdir=tests/ --xdoctest xclim
+commands_post =
```

Comment:
New empty directive ? Does this do something ?

Reply:
This is a feature in tox: Since this recipe is based on testenv, we can overwrite an existing directive to have nothing by simply creating an empty directive.

Why do that here? Because we can't run `pytest --nbval` with coverage (causes errors) so we don't want to bother with uploading coverage in case a user accidentally tries to. It already won't let them (`notebooks` doesn't accept the `coverage` modifier) but this is extra security.